pkg/cloudinit/common.go

```diff
@@ -9,9 +9,26 @@ import (
 	"k8s.io/apimachinery/pkg/util/version"
 )
 
+type InstallOption string
+
+const (
+	InstallOptionChannel   InstallOption = "track"
```

Comment:
I'm a bit confused about this part. I can see `channel` instead of `track` and `localPath` instead of `local-path`.
https://github.com/canonical/cluster-api-k8s/blob/main/pkg/ck8s/workload_cluster.go#L279-L288

If these are the same, maybe we can centralize them?

Reply:
I've updated the `track` mentions to `channel`. For `localPath` vs `local-path`, I've kept `local-path` to fit the separate with `-` file naming convention we had. I'll adjust the `k8s-snap-api` to fit this form with another (cleanup) card I've this pulse.